tests/creature_test.cpp

```diff
@@ -20,6 +20,16 @@ void calculate_bodypart_distribution( monster &attacker, monster &defender, int
         selected_part_histogram[defender.select_body_part(&attacker, hit_roll)]++;
     }
 
+    if( hit_roll != 0 ) {
```

Comment:
Instead of copying the game formula, I'd rather generate a table of distributions based on a handful of hit rolls (0, 1, 100, maybe a few in between), it nicely illustrates what the goal is by showing what the distributions are.